pkg/ingester/transfer.go

```diff
@@ -377,16 +381,30 @@ func (i *Ingester) TransferOut(ctx context.Context) error {
 		MaxRetries: i.cfg.MaxTransferRetries,
 	})
 
+	// Keep track of the last error so that we can log it with the highest level
+	// once all retries have completed
+	var err error
+
 	for backoff.Ongoing() {
-		err := i.transferOut(ctx)
+		err = i.transferOut(ctx)
 		if err == nil {
 			return nil
 		}
 
-		level.Error(util.Logger).Log("msg", "transfer failed", "err", err)
+		// Log just with an info level errors which are expected to happen
+		var lvl log.Logger
+		if go_errors.Is(err, errTransferNoPendingIngesters) {
+			lvl = level.Info(util.Logger)
```

Comment:
Let's just log them all as warnings, and last one as error? This code seems to be too smart.

Reply:
Probably. What's about now?

Reply:
I think that's fine. Thanks.